### DISCUSSION TITLE
Deprecation Treatment of Jaeger Exporter

### DISCUSSION THREAD

**Discussion by hdost** [open-telemetry/opentelemetry-rust#1341] [General]
As was recently brought up https://github.com/open-telemetry/opentelemetry-rust/pull/1323#issuecomment-1792886516 we know that [Jaeger is deprecated](https://github.com/open-telemetry/opentelemetry-specification/pull/2858). 

[Jaeger Exporter](https://crates.io/crates/opentelemetry-jaeger) Max Daily download 14k Downloads in the last 90 days
[OTLP](https://crates.io/crates/opentelemetry-otlp) by comparison it's max daily >40k 

So from that we have a non-insignificant portion of users who are currently using jaeger. 

This puts forward a couple of points to address. 

1. As we discussed at the SIG we will not look to stabilize Jaeger itself, but in service to our users we should consider keeping at least getting it to be compatible with our 1.x of api and sdk. The main reason being that we need to remember the developers writing the code don't always have control over the environment in which they run. 
2. We should at least until we're sdk stable keep jaeger in the main repo and we could later consider moving it out to contrib repo when it exists. #841
3. We still don't have a release to note that `opentelemetry-jaeger` is deprecated. When we do we should make sure we have a migration guide showing people how they can move. 
4. How do we handle changes coming in?  My suggestion would be, we no longer accept feature requests and PRs because we don't have the coverage to guarantee things won't currently break on it. We should accept and update as mentioned to get compatibility with 1.x
5. We should potentially at least ensure that it fully works with 1.x meaning also have an integration test. It wouldn't be far off from #1327 and once the first one is done the rest should be very similar.

**Comment by cijothomas** [comment]
Thanks for opening this. My GH notifications was broken, so did not see this!

https://github.com/open-telemetry/opentelemetry-rust/pull/1345 One small step in the direction of showcasing OTLP can be used instead of jaeger-exporter!

**Reply by hdost** [reply]
I realized that https://github.com/open-telemetry/opentelemetry-rust/issues/995 already exists i apparently didn't do good enough issue searching.

**Comment by hdost** [comment]
Continue conversation in https://github.com/open-telemetry/opentelemetry-rust/issues/995
